packages/contracts/docs/platform.md

```diff
@@ -0,0 +1,189 @@
+# Smart Contracts Documentation
+
+## Overview
+
+This documentation provides details on the smart contracts used in the system, including their properties and methods. It also includes integration notes for connecting these contracts with other components like `maci-subgraph`.
+
+## Contracts
+
+### Contract: `EASRegistryManager`
+
+The `EASRegistryManager` contract handles requests for registry management. Users send requests to this contract, and a coordinator approves or rejects these requests.
+
+#### Properties
+
+- **contractId**: `uint256` - Unique identifier for the contract.
```

Comment:
this should be the address of the EAS Registry contract, right? then I think we could specify it instead of just say it's a unique identifier for the contract?